modules/core/src/main/java/org/apache/synapse/ServerManager.java

```diff
@@ -444,6 +445,7 @@ private void doShutdown() {
 
             // mark as destroyed
             changeState(ServerState.UNDETERMINED);
+            OpenTelemetryManagerHolder.getOpenTelemetryManager().close();
```

Comment:
It can be null when the exporter is null. But in here(for jaeger and zipkin) it won't be null unless required bundles are absent. For custom tracing there's a chance of being null if exporter is not set correctly. Will add a condition to check and if so do the operation.